server/fishtest/templates/tests_view.mak

```diff
@@ -2,6 +2,8 @@
 
 <%!
   import json
+  args_row_count = 0
+  args_row_height = "33px"
```

Comment:
Ooops, it seems that I was wrong, please revert this change, it seems that in that way the names become const. 

```yaml
May 27 15:45:29 dfts-0 pserve[27882]:   File "/home/usr00/fishtest/server/fishtest/templates/tests_view.mak", line 80, in render_body
May 27 15:45:29 dfts-0 pserve[27882]:     <% args_row_count += 1 %>
May 27 15:45:29 dfts-0 pserve[27882]: UnboundLocalError: cannot access local variable 'args_row_count' where it is not associated with a value
```